src/test/kotlin/study/StringCalculatorTest.kt

```diff
@@ -0,0 +1,77 @@
+package study
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.AssertionsForClassTypes
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EmptySource
+import org.junit.jupiter.params.provider.NullSource
+
+class StringCalculatorTest {
+    @DisplayName("입력 값 테스트 NULL  혹은 빈문자열인경우")
+    @ParameterizedTest
+    @NullSource
+    @EmptySource
```

Comment:
```suggestion
    @NullAndEmptySource
```

두 조건은 하나로 합칠 수 있습니다 ㅎㅎ